contrib/opencensus-ext-pymongo/tests/test_pymongo_trace.py

```diff
@@ -124,8 +136,14 @@ def start_span(self, name=None):
         span = mock.Mock()
         span.name = name
         span.attributes = {}
+        span.status = {}
         self.current_span = span
         return span
 
     def add_attribute_to_current_span(self, key, value):
         self.current_span.attributes[key] = value
+
+    def set_status_to_current_span(self, code, message, details):
```

Comment:
We might want to use `get_current_span` to get the span and use `Span.set_status`, instead of introducing a new API for Tracer. The fact that we had `add_attribute_to_current_span` has been corrected by [OpenTelemetry spec](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/api-tracing.md#getcurrentspan).

Reply:
So `add_attribute_to_current_span` should be removed in the future? Will change `set_status` now.

Reply:
Shouldn't `current_span` be renamed to `get_current_span` following the spec? Or the name is not mandatory?

Reply:
We will use the correct name in OpenTelemetry https://github.com/open-telemetry/opentelemetry-python/blob/master/opentelemetry-api/src/opentelemetry/trace/__init__.py#L208.

For OpenCensus, we cannot break the existing API unless we move to the next major version (which is OpenTelemetry).

Reply:
> For OpenCensus, we cannot break the existing API unless we move to the next major version (which is OpenTelemetry).

Indeed

Reply:
Fixed as requested! Please take a look!